### ISSUE TITLE
Streamline input name mismatch in transformed matmul nodes

### ISSUE THREAD

**Issue by AndreaRigoni** [Xilinx/finn#43]
The Streamline transform generates strange names in inputs of the transformed MatMul nodes that are actually missing in the graph causing a disconnected network. 
For instance the following code generate a disconnected graph:
```python
    lfc = LFC(weight_bit_width=1, act_bit_width=1, in_bit_width=1)
    checkpoint = torch.load(trained_lfc_checkpoint, map_location="cpu")
    lfc.load_state_dict(checkpoint["state_dict"])
    bo.export_finn_onnx(lfc, (1, 1, 28, 28), export_onnx_path)
    model = ModelWrapper(export_onnx_path)
    model = model.transform(InferShapes())
    model = model.transform(FoldConstants())
    model = model.transform(GiveUniqueNodeNames())
    model = model.transform(GiveReadableTensorNames())
    model = model.transform(Streamline())
```
the corresponding net shown in netron is:
![image](https://user-images.githubusercontent.com/1974299/74259145-b5dcc900-4cf7-11ea-8aa2-180037f74595.png)

The problem seems to happen in BatchNormToAffine()

**Comment by AndreaRigoni** [comment]
My fault .. I didn't get that some of the operands are not correctly shown in netron .. maybe because they are not correctly exported to onnx.

**Comment by maltanar** [comment]
Hi Andrea, can you verify that this issue is resolved after PR #44 is merged?

**Comment by AndreaRigoni** [comment]
Hi Yaman, verified ! Sorry, i missed to close the issue.
